docs/source-pytorch/conf.py

```diff
@@ -110,6 +110,11 @@ def _transform_changelog(path_in: str, path_out: str) -> None:
     target_dir="docs/source-pytorch/integrations/hpu",
     checkout="tags/1.0.0",
 )
+assist_local.AssistantCLI.pull_docs_files(
+    gh_user_repo="Lightning-AI/lightning-Graphcore",
+    target_dir="docs/source-pytorch/integrations/ipu",
+    checkout="tags/0.1.0.rc3",
```

Comment:
```suggestion
    checkout="tags/0.1.0",
```
This expect to have first final release, do not stick with RC only